documentation/modules/accessing-logs-ui.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="accessing-logs-ui_{context}"]
+= Downloading logs from the web console
+
+You can download logs for a completed, failed, or canceled migration plan and for migrated virtual machines (VMs) by using the {project-short} web console.
+
+.Procedure
+
+. In the web console, click *Migration plans*.
+. To download logs for a migration plan, click *Get logs* beside the migration plan name.
+. To download logs for a migrated VM, click the migration plan name and then click *Get logs* beside the VM.
++
+You can download the VM logs while the migration plan is running.
+
+. In the *Get logs* window, click *Get logs*.
++
```

Comment:
- Once pressing on Get logs in the Get logs window, the logs are being collected and a message pops up once it is done:
"Log collection complete."
- 'Download logs' button replaces the 'Get logs' button and the user have to press on it to download the logs (archived file)

Reply:
I was using Vince's mockups. I will check the forklift-ui repo